pykokkos/lib/ufunc_workunits.py

```diff
@@ -1,6 +1,355 @@
 import pykokkos as pk
 
 
+@pk.workunit
+def multiply_impl_1d_double(tid: int, viewA: pk.View1D[pk.double], viewB: pk.View1D[pk.double], out: pk.View1D[pk.double], ):
+    out[tid] = viewA[tid] * viewB[tid % viewB.extent(0)]
```

Comment:
the modulo operator is leftover from before here--it seems harmless in local testing but should probably be removed (though it may have some merit depending on how broadcasting evolves I suppose..)